mips_api/__init__.py

```diff
@@ -362,9 +362,10 @@ def process_chart(params, chart_dict, omit_list, other, no_program):
     priority_codes = _param_priority_list(params)
 
     # add short codes
-    for code, name in chart_dict.items():
+    for code, _name in chart_dict.items():
         if len(code) >= code_len:
             short = code[:6]  # ignore the last two digits on active codes
+            name = _name.replace("(", "").replace(")", "")  # remove parentheses
```

Comment:
hmm, I'm wondering whether taking this approach is better than just nudging finance to not use chars that AWS doesn't like?  This seems like slippery slope, if we take this approach then should we not do this for the entire set of invalid character pattern `{^([\p{L}\p{Z}\p{N}_.:/=+\-@]*)$}]`?

Reply:
Nudging finance to not use parentheses because it would break our deploy pipeline seems like a bad idea to me, instead we should make sure that finance isn't able to break our deploy pipeline with a new program name. And yeah, it's probably better to just apply the same regex and remove any invalid characters, along with enforcing the 256 character limit for tag values.

https://docs.aws.amazon.com/mediaconnect/latest/ug/tagging-restrictions.html

Reply:
fair argument